src/routes/TriggerMessage/container.js

```diff
@@ -8,15 +8,16 @@ import TriggerMessage from './component';
 
 
 export class TriggerMessageContainer extends Component {
-	handleStart() {
+	handleStart(props) {
 		parentCall('setFullScreenDocumentMobile');
 		parentCall('openWidget');
+		props.onRestore();
 		route('/');
 	}
 
 	render = (props) => {
 		parentCall('resetDocumentStyle');
-		return <TriggerMessage onStartChat={this.handleStart} {...props} />;
+		return <TriggerMessage onStartChat={() => this.handleStart(props)} {...props} />;
```

Comment:
any need for this change? 👀

Reply:
yes, we needed to add an input value to the handleStart function